src/DDTrace/Integrations/PDO.php

```diff
@@ -0,0 +1,230 @@
+<?php
+
+namespace DDTrace\Integrations;
+
+use DDTrace\Tags;
+use DDTrace\Types;
+use OpenTracing\GlobalTracer;
+
+class PDO
+{
+    private static $connections = [];
+    private static $statements = [];
+
+    /**
+     * Static method to add instrumentation to PDO requests
+     */
+    public static function load()
+    {
+        if (!extension_loaded('ddtrace')) {
+            trigger_error('ddtrace extension required to load PDO integration.', E_USER_WARNING);
+            return;
+        }
+
+        // public PDO::__construct ( string $dsn [, string $username [, string $passwd [, array $options ]]] )
+        dd_trace('PDO', '__construct', function (...$args) {
```

Comment:
While using `PDO::_construct` to setup necessary data to be used later i.e. `PDO::storeConnectionParams($this, $args);` is fine. However I wonder if there is any added value from creating a span around a constructor. 
Is this constructor doing some IO ?

Reply:
Yes, it actually connects when you call __construct, so this span captures connection time as well as potential connection errors:

![screen shot 2018-10-10 at 1 06 59 pm](https://user-images.githubusercontent.com/52718/46753313-7f1f1a80-cc8d-11e8-9130-4ab900f46ef8.png)